public/manifest.json

```diff
@@ -0,0 +1,8 @@
+{
+	"short_name": "",
+	"name": "",
+	"start_url": ".",
+	"display": "standalone",
+	"theme_color": "#000000",
+	"background_color": "#ffffff"
```

Comment:
use var

Reply:
I have not added these It's already come packed with the library itself